### PR TITLE
[DOCS] Add a Procedure for Adding a Validation to a Checkpoint to the GX Cloud Docs

### PR DESCRIPTION
**Pull request by kwcanuck**
[DOC-673](https://greatexpectations.atlassian.net/browse/DOC-673) requests the addition of a procedure for adding a Validation to a Checkpoint in GX Cloud.  This PR implements this change. 

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

[DOC-673]: https://greatexpectations.atlassian.net/browse/DOC-673?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Shinnnyshinshin — APPROVED** [review]
Looks great :) thank you for this change @kwcanuck

File: docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md [hunk 1]
```diff
@@ -81,6 +81,40 @@ To learn more about Checkpoints, see [Checkpoint](/reference/learn/terms/checkpo
...
+    ```python title="Jupyter Notebook"
+        validations = [
+        {
+            "batch_request": {
+                "datasource_name": "your_datasource_name",
+                "data_asset_name": "your_data_asset_name",
+            },
+            "expectation_suite_name": "your.expectation.suite.name",
+        },
+    ]
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md:106]: This should be pulled from a snippet in an  example script under test.

File: docs/docusaurus/versioned_docs/version-0.18/cloud/checkpoints/manage_checkpoints.md [hunk 2]
```diff
@@ -81,6 +81,40 @@ To learn more about Checkpoints, see [Checkpoint](/reference/learn/terms/checkpo
...
+        validations = [
+        {
+            "batch_request": {
+                "datasource_name": "your_datasource_name",
+                "data_asset_name": "your_data_asset_name",
+            },
+            "expectation_suite_name": "your.expectation.suite.name",
+        },
+    ]
+    ```
```
Comment (Rachel-Reverie) [inline docs/docusaurus/versioned_docs/version-0.18/cloud/checkpoints/manage_checkpoints.md:107]: This should also be pulled from an example script under test.

File: docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md (hunk 1 above)
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md:106]: @Rachel-Reverie as discussed, what you see is a temporary solution (I hope). In the future, running code should be unnecessary.

File: docs/docusaurus/versioned_docs/version-0.18/cloud/checkpoints/manage_checkpoints.md (hunk 2 above)
Comment (kwcanuck) [reply docs/docusaurus/versioned_docs/version-0.18/cloud/checkpoints/manage_checkpoints.md:107]: @Rachel-Reverie, as discussed, what you see is a temporary solution (I hope). In the future, running code should be unnecessary.

**Pull request merged**